### PR TITLE
Fixes #3219

### LINKED ISSUE #3219: URL to Android Material Design guidelines in CONTRIBUTING.md redirects to a 404 page
There is a url in [CONTRIBUTING.MD](https://github.com/TeamAmaze/AmazeFileManager/blob/release/3.7/CONTRIBUTING.md) that linked to Android Material Design guidelines, which is https://material.io/guidelines/material-design/introduction.html, currently it redirects to https://material.io/design/material-design/introduction.html and finally becomes a 404 page.

There are links in their document with similar meaning, not sure if they may help.

https://material.io/design/introduction
https://material.io/design/guidelines-overview

### PR DESCRIPTION
**Pull request by CPHafis**
Fixes #3219

Replaced the old link for introduction to material design to new working link

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Please keep in mind the points below before considering contributing to Amaze:
 You won't be able to claim the license for changes made by you unless you do that. 
 If there's no license header in any file, please include one from [GNU GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html) webpage.
 - Please follow [Android/JAVA code style](https://source.android.com/source/code-style.html) for writing any code, but do not use the Hungarian notation proposed - discussion [here](https://github.com/TeamAmaze/AmazeFileManager/issues/986). 
-Also, follow [Android Material Design guidelines](https://material.io/guidelines/material-design/introduction.html) in case you make changes to any UI element.
+Also, follow [Android Material Design guidelines](https://m2.material.io/design/introduction) in case you make changes to any UI element.
```
Comment (VishnuSanal) [inline CONTRIBUTING.md:13]: I think you can replace it with the latest material design website (https://m3.material.io/get-started)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,4 +39,4 @@ Finally:
 
 If we feel your PR is a significant help to us, we'll award you a bounty with any of your preferred mode of payment.  
 Please provide the details for the same once asked.
-Ready to roll? Start forking ;)
\ No newline at end of file
+Ready to roll? Start forking ;)
```
Comment (VishnuSanal) [inline CONTRIBUTING.md:42]: please revert this line :)

**Review by VishnuSanal — CHANGES_REQUESTED** [review]
@CPHafis good work, please take a look at the comments

File: CONTRIBUTING.md (hunk 1 above)
Comment (CPHafis) [reply CONTRIBUTING.md:13]: ok

File: CONTRIBUTING.md (hunk 2 above)
Comment (CPHafis) [reply CONTRIBUTING.md:42]: ok

**Review by VishnuSanal — APPROVED** [review]

**Review by VishalNehra — APPROVED** [review]

**Pull request merged**